tcms/settings/common.py

```diff
@@ -105,7 +105,8 @@
     (_('TELEMETRY'), [
         (_('Testing'), [
             (_('Breakdown'), reverse_lazy('testing-breakdown')),
-            (_('Status matrix'), reverse_lazy('testing-status-matrix'))
+            (_('Status matrix'), reverse_lazy('testing-status-matrix')),
+            (_('Execution trends'), reverse_lazy('testing-execution-trends'))
```

Comment:
add a comma at the end so you don't have to edit the previous line the next time.

Reply:
^^^